docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -132,54 +132,18 @@ The following tables provide resource requirements and scaling guidelines based
 on the number of agents required by your deployment:
 
 * <<resource-requirements-by-number-agents>>
-* <<recommend-settings-scaling-agents>>
 
 [discrete]
 [[resource-requirements-by-number-agents]]
 === Resource requirements by number of agents
 |===
 | Number of Agents | Memory       | vCPU            | {es} Cluster size
```

Comment:
Can we also clarify that Memory and vCPU are for Fleet Server?

Reply:
yep good point done.